sql/moz-fx-data-shared-prod/monitoring/bigquery_usage_costs/view.sql

```diff
@@ -5,6 +5,7 @@ SELECT
   creation_date,
   source_project,
   user_email,
+  username,
```

Comment:
do we need username if we already have user_email?

Reply:
The two columns capture different information.  Username is extracted from the query and user_email captures service accounts and emails.